dotcom-rendering/src/components/ArticleMeta.web.tsx

```diff
@@ -341,8 +379,17 @@ export const ArticleMeta = ({
 								<Avatar src={avatarUrl} alt={authorName} />
 							</MetaAvatarContainer>
 						)}
-
 						<div>
+							{isAudio && podcastTag && (
+								<PodcastMeta
+									image={podcastTag.image}
+									spotifyUrl={podcastTag.spotifyUrl}
+									subscriptionUrl={podcastTag.subscriptionUrl}
+									audioDownloadUrl={audioDownloadUrl}
+									rssFeedUrl={rssFeedUrl}
+								/>
+							)}
```

Comment:
I wish there was a better way of doing this, rather than having all this audio-specific logic defined in the top `ArticleMeta` component. Unhelpful that I can't think of anything better 😅

Reply:
This will be refactored out when we create the audioLayout layout It just works for now where we are using standard Layout